composer.json

```
@@ -0,0 +1,32 @@
+{
+	"name": "wordpress/twentytwentytwo",
+    "type": "package",
```

There is a mix of tabs and spaces here, we should likely go with spaces since this is `json`. This corrects the first key but there may be others to check as well.

```suggestion
    "name": "wordpress/twentytwentytwo",
    "type": "package",
```